laws/shared/src/test/scala/algebra/laws/LawTests.scala

```diff
@@ -89,13 +89,7 @@ trait LawTestsBase extends FunSuite with Discipline {
   laws[RingLaws, BigInt].check(_.euclideanRing)
 
   {
-    implicit val tupEq: Eq[(Int, Int)] = new Eq[(Int, Int)] {
-      def eqv(a: (Int, Int), b: (Int, Int)) = a == b
-    }
-    implicit val band = new Band[(Int, Int)] {
-      def combine(a: (Int, Int), b: (Int, Int)) = (a._1, b._2)
-    }
-    checkAll("(Int, Int) Band", GroupLaws[(Int, Int)].band)
```

Comment:
why did you delete this band test? If nothing else it was exercising GroupLaws.band.